### ISSUE TITLE
Controls under SQL API section are not adapting number assignation using dragon speech recognition tool.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1023]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1104089

**Comment by ato9000** [comment]
Repro Steps:
1. https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/79003cad-e368-4304-acbf-3d3aa80978e3/resourceGroups/appcenter-data-ForScenario-Creation/providers/Microsoft.DocumentDB/databaseAccounts/abc/dataExplorer

2. Navigate to Azure Cosmos DB. and select it
3. Select any created Azure Cosmos DB and navigate to Data explorer and select it.
4. Navigate to SQL API blade and navigate to all items as "Query" and expand it.
5. Navigate to all control under Query option.
6. Verify Control type of  control using developer tool(F12) and verify that these are adapting number assignation or not.



Actual result:
Controls under  SQL API section are not adapting number series using dragon speech recognition tool. Controls like " Cosmos, Items etc." are not defined programmatically. 
While navigating using Screen reader on "Items" screen reader announcing only "Item Group" it does nota announce as button list item etc.

Expected Result:
Controls under  SQL API section should be adapt number series using dragon speech recognition tool. Controls like " Cosmos, Items etc." should defined programmatically. 



Note:
This issue also exists with Firefox + NVDA/JAWS.
This issue also exists with Dragon speech recognition tool.

![image](https://user-images.githubusercontent.com/8519407/132721345-ec90994b-782e-493d-9ef9-90486a6fcb4d.png)

https://user-images.githubusercontent.com/8519407/132721461-7b7bd153-8cd7-4681-99c8-8aaa568f5446.mp4
